README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Build Status](https://api.travis-ci.org/haskell-nix/hnix.svg)](https://travis-ci.org/haskell-nix/hnix)
 [![Chat](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/haskell-nix/Lobby)
-<sup>([Hackage Matrix Builder](https://matrix.hackage.haskell.org/package/hnix))</sup>
+[![Hackage Matrix Builder](https://img.shields.io/badge/Hackage%20Matrix-Builder-green)](https://matrix.hackage.haskell.org/package/hnix)
```

Comment:
Is there a version of this badge that indicates the actual build status? It's a bit funny to have a green badge when the latest Matrix report is quite red. ;)

Reply:
Of course I checked are there Hackage Build Matrix budges!

There is none.

To be a mad scientist - it is possible to learn Hackage Build Matrix API and somehow build a custom hook that generates badge. But anyway - it is better to post a matrix on the front page, but there is no such possibility also.

Reply:
Thanks! I'd slightly prefer a less suggestive color like maybe blue. But I don't care too much.

Reply:
There are some injection possibilities in GitHub.

Badges themself exploit that.

So it is theoretically possible to get the Hackage Build Matrix itself and inject it as for example SVG.

When I played with things like that (4-6 years ago) - SVG support on GitHub wished to be better.

Reply:
But that is a small, clear free `project` in itself to share to the Haskell community.

Reply:
Oh, I just remembered that we actually have 2! GSoC projects related to Hackage badges: https://summerofcode.withgoogle.com/organizations/6387985961975808

So maybe we can adopt a more informative badge at the end of the summer. :)

Reply:
Hackage API is great.

Injecting Hackage Build Matrix, can be hacked together in 4 hours.

Just cut the HTML table there and convert it into SVG, automate SVG update, and inject it into the page.

Until Hackage does not change the design - that would work.

But API and ready badge/injection arel of course, preferable.

There probably even some good GitHub projects for assembling&automating PNG/SVG injection.

Reply:
But the laziness is preferable.